newrelic-agent/src/test/java/com/newrelic/api/agent/NewRelicApiImplementationTest.java

```diff
@@ -77,6 +78,22 @@ public void noticeErrorShouldAcceptNumberAndBooleanAttributeType() {
 
     }
 
+    @Test
+    public void noticeErrorShouldEvaluateExpectedClassesConfiguration(){
+        //This is a wip.  I need to figure out how to check the config/have mockito
+        //provide a way to set the errorCollectorConfig.
+        //Leaving in as guarantee pass as a place-marker.
+        mockOutServices();
+        Map<String, Object> expectedValues = new HashMap<>();
+        String msg = "this should not be expected";
+
+        expectedValues.put("java.lang.Throwable", msg);
+        NewRelicApiImplementation target = new NewRelicApiImplementation();
+        target.noticeError(msg);
+        Mockito.verify(errorService).reportError(msg, Collections.EMPTY_MAP, false);
+
+    }
+
```

Comment:
Turns out all of this is well tested in the APITest class.  I think I'm going to to end up dropping this.